tests/generate.spec.tsx

```diff
@@ -6,6 +6,7 @@ import luxonGenerateConfig from '../src/generate/luxon';
 import { getMoment } from './util/commonUtil';
 
 import 'dayjs/locale/zh-cn';
+import 'dayjs/locale/ko';
```

Comment:
Seems change in `generate/dateFns.ts` should not affect dayjs, is this import necessary?

Reply:
dayjs locale js file should be imported before use.
cause of dynamic register locale config

Reply:
The problem is that the fix for dateFNS should not be covered using dayjs test cases.